scripts/deploysnapshot.sh

```diff
@@ -21,6 +21,7 @@ SNAPSHOTDIR=/var/www/vhosts/mf-geoadmin3/private/snapshots/$1
 # Prepare snapshot
 cd $SNAPSHOTDIR/geoadmin/code/geoadmin
 if [ "$NEW_VERSION" = "true" ]; then
+  export KEEP_VERSION='false'
```

Comment:
I'm not sure to understand what is the difference between NEW_VERSION and KEEP_VERSION, why don't we put KEEP_VERSION =false in rc_(dev|int|prod) directly and remove NEW_VERSION ?